migrations/versions/666003748d14_change_active_nullable_and_default.py

```diff
@@ -0,0 +1,43 @@
+"""change_active_nullable_and_default
+
+Revision ID: 666003748d14
+Revises: a5a40d843415
+Create Date: 2022-02-24 16:35:34.228040
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+from sqlalchemy.orm.session import Session
+from dds_web.database import models
+
+
+# revision identifiers, used by Alembic.
+revision = "666003748d14"
+down_revision = "a5a40d843415"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    session = Session(bind=op.get_bind())
+    all_user_rows = session.query(models.User).all()
+    for user in all_user_rows:
+        if user.active is None:
+            user.active = False
+    session.commit()
+    op.alter_column("users", "active", existing_type=mysql.TINYINT(display_width=1), nullable=False)
```

Comment:
Not sure: should the new default value be mentioned here, or is that only in the model?

Reply:
It seems like it. Not adding / looking into this now though